e2e/app/component/workspace-card.ts

```diff
@@ -158,15 +158,18 @@ export default class WorkspaceCard extends CardBase {
    * @param {boolean} waitForDataPage Waiting for Data page load and ready after click on Workspace name link.
    */
   async clickWorkspaceName(waitForDataPage = true): Promise<string> {
-    const [elemt] = await this.asElementHandle().$x(`.//*[${WorkspaceCardSelector.cardNameXpath}]`);
-    const name = await getPropValue<string>(elemt, 'textContent');
-    await Promise.all([
-      this.page.waitForNavigation({ waitUntil: ['domcontentloaded', 'networkidle0'] }),
-      elemt.click()
-    ]);
```

Comment:
The `Promise.all` should be inside of the `if (waitForDataPage)` because navigation does not happen when `waitForDataPage` is `false`.